cluster/images/Dockerfile

```diff
@@ -16,6 +16,8 @@ FROM alpine:3.19.1
 
 ARG BINARY
 
+LABEL maintainer="karmada.io"
```

Comment:
@B1F030 Can `LABEL` refer to `REGISTRY` and `VERSION` as the input environment variables of `docker.sh` , so that different scenarios can be configured as needed, and the scope of the label can also be reduced.

```shell
function build_local_image() {
  local -r output_type=$1
  local -r target=$2
  local -r platform=$3

  local -r image_name="${REGISTRY}/${target}:${VERSION}"

  echo "Building image for ${platform}: ${image_name}"
  set -x
  docker build --build-arg BINARY="${target}" \
          ${DOCKER_BUILD_ARGS} \
          --tag "${image_name}" \
          --file "${REPO_ROOT}/cluster/images/Dockerfile" \
          --label "${LABEL}"\
          "${REPO_ROOT}/_output/bin/${platform}"
  set +x

  if [[ "$output_type" == "registry" ]]; then
    docker push "${image_name}"
    signImage ${image_name}
  fi
}
```

Reply:
What about now?
And in my humble opinion, we should clean dangling images each run, so no need to match the label exactly in docker prune.

Reply:
Updated to `--label "version.karmada.io=${REGISTRY}:${VERSION}"`.